formula/src/test/java/com/instacart/formula/internal/TestInspector.kt

```diff
@@ -20,14 +20,6 @@ class TestInspector : Inspector {
         events.add("evaluate-finished: ${formulaType.qualifiedName}")
     }
 
-    override fun onExecutionStarted() {
-        events.add("execution-started")
-    }
-
-    override fun onExecutionFinished() {
-        events.add("execution-finished")
-    }
-
     override fun onActionStarted(formulaType: KClass<*>, action: DeferredAction<*>) {
         events.add("action-started: ${formulaType.qualifiedName}")
```

Comment:
Do you expect that outside consumers will be reading these events? if so, maybe pulling `action-started` and others out into a const so that consumers can check for these values would be nice

Reply:
`TestInspector` is not part of the library, it's just within formula tests.